aegis_director/aegis_director/main.py

```diff
@@ -0,0 +1,96 @@
+import time
+import numpy as np
+
+import rclpy
+from aegis_director.robot_director import RobotDirector
+from aegis_director.utils import quaternion_to_euler, euler_to_quaternion
```

Comment:
Hasn't that been already imported in _init_ file?

Reply:
Yep, that's true, we can simplify this to:
```suggestion
from aegis_director import RobotDirector, quaternion_to_euler, euler_to_quaternion
```